src/workerd/api/worker-rpc.c++

```diff
@@ -2,19 +2,375 @@
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
 
-#pragma once
-
 #include <workerd/api/worker-rpc.h>
+#include <workerd/api/global-scope.h>
 
 namespace workerd::api {
 
+namespace _ { // private
```

Comment:
This can be an anonymous namespace since it's only used within this source file.

```suggestion
namespace {
```